web_responsive/static/src/components/apps_menu/apps_menu.esm.js

```diff
@@ -34,6 +34,12 @@ export class AppsMenu extends Component {
         this.state = useState({open: false});
         useBus(this.env.bus, "ACTION_MANAGER:UI-UPDATED", () => {
             this.setState(false);
+            var attachmentViewerCloseButton = document.querySelector(
```

Comment:
In other code it uses: `this.messaging.dialogManager` to access mail module related components. But I don't know how to get to there maybe with `registerMessagingComponent` so this solution is fine

Reply:
Hey @tarteo , made some updates based on your suggestion!!